app/views/media/index.html.erb

```diff
@@ -0,0 +1,86 @@
+<!-- javascript requirements -->
+<%= javascript_include_tag 'application' %>
+<script sync src="https://platform.twitter.com/widgets.js"></script>
+
+<!-- Flash notifications -->
+<% if !flash[:error].nil? %>
+  <div class="error-message-home-pg">
+    <strong>You can't do that!</strong> <%= flash[:error] %>.
+  </div>
+<% end %>
+
+<% if !current_spy.nil? %>
+  <% if @marks.empty? %>
+    <h1 class="page-header">NO MARKS YET <%= fa_icon "crosshairs" %> </h1>
+  <% else %>
+    <div class="container">
+      <!-- Header -->
+      <div class="page-header">
+        <h1>TRACK YOUR MARKS</h1>
+      </div>
+
+      <div class="filter">
+        <%= link_to root_path(
+            params: {
+              filter: "vimeo"
+            }
+          ) do %>
+            <%= fa_icon "vimeo", class: "filter-button", alt: "sort by vimeo" %>
+        <% end %>
+
+        <%= link_to root_path(
+            params: {
+              filter: "twitter"
+            }
+          ) do %>
+            <%= fa_icon "twitter", class: "filter-button", alt: "sort by twitter" %>
+        <% end %>
+
+        <%= link_to refresh_feed_path do %>
+          <%= fa_icon "refresh", class: "filter-button", alt: "refresh" %>
+        <% end %>
+
+      </div>
+      <ul class="timeline">
+        <% media_count = @media.length %>
+
+        <% @media.each do |m| %>
+
+          <% if media_count % 2 == 0 %>
```

Comment:
This would also be good to extract into a helper method. A rule of thumb is, don't do math on your view. Do it elsewhere and call *that* method in the view.